.github/workflows/test.yml

```diff
@@ -58,33 +55,34 @@ jobs:
           #   ghc: '8.6.4'
 
     steps:
-      # Cancel queued workflows from earlier commits in this branch
-      - uses: fkirc/skip-duplicate-actions@master
```

Comment:
I think we still need this step to skip unnecessary jobs started by outdated commits

Reply:
mmm does not the same action in the `pre_job` do all the needed work? Afaiu it does all the things and no depends on the params: https://github.com/fkirc/skip-duplicate-actions#usage-examples

Reply:
The log of a build in my branch can provide some evidence:

```
Did not find other workflow-runs to be cancelled
Stop backtracking at commit https://github.com/jneira/haskell-language-server/commit/3eddabf804b4a09ea07cff26f5470514b1639f50 because '.github/workflows/test.yml' are not skippable against paths '' or paths_ignore '**/docs/**,**.md,**/LICENSE,.circleci/**,install/**,nix/**'
Do not skip execution because we did not find a transferable run
```

https://github.com/jneira/haskell-language-server/pull/22/checks?check_run_id=2256857741

Reply:
Oh, does it work per workflow? I thought it's per job...